core/trino-main/src/test/java/io/trino/execution/TestSqlTaskExecution.java

```diff
@@ -632,6 +632,7 @@ private PartitionedOutputBuffer newTestingOutputBuffer(ScheduledExecutorService
                 Thread.sleep(10);
             }
             catch (InterruptedException e) {
+                Thread.currentThread().interrupt();
                 // do nothing
```

Comment:
do nothing?

Maybe this method could be replaced with `io.trino.testing.assertions.Assert#assertEventually`?